analyzers/src/SonarAnalyzer.CSharp/Rules/NonDerivedPrivateClassesShouldBeSealed.cs

```diff
@@ -24,48 +24,80 @@ namespace SonarAnalyzer.Rules.CSharp
     public sealed class NonDerivedPrivateClassesShouldBeSealed : SonarDiagnosticAnalyzer
     {
         private const string DiagnosticId = "S3260";
-        private const string MessageFormat = "Private classes or records which are not derived in the current assembly should be marked as 'sealed'.";
+        private const string MessageFormat = "{0} {1} which are not derived in the current assembly should be marked as 'sealed'.";
```

Comment:
Maybe use "classes" and "record classes"? It shouldn't raise for structs or record structs but maybe it would be better to be specific.